### ISSUE TITLE
Bug

### ISSUE THREAD

**Issue by realswikarrr** [Sajani55/Content-Management-System#15]
![1](https://user-images.githubusercontent.com/80502023/117991229-93ec4f00-b35d-11eb-96e5-b5402ca33432.PNG)
![2](https://user-images.githubusercontent.com/80502023/117991245-96e73f80-b35d-11eb-8a57-817b6047552f.PNG)

@thebuggybug

**Comment by Sajani55** [comment]
The alignment of both gallery and testimonial section doesnot looks good. The devlopers will fix this.

**Comment by thebuggybug** [comment]
The issue with grid has been solved. All we need to do is replace some lines of backend code. 
start **Starpi Backend** code after  `<MDBRow> `tag to function it properly.
for fxing all grid issue, this can be done to solve.
![image (2)](https://user-images.githubusercontent.com/68284654/118071938-802a0280-b3c8-11eb-88e9-7a6fd9f85bf0.png)
![image (3)](https://user-images.githubusercontent.com/68284654/118071951-88823d80-b3c8-11eb-8eeb-481af5db74d1.png)


![image (13)](https://user-images.githubusercontent.com/68284654/118071859-5b358f80-b3c8-11eb-9374-ddeec2fda9e9.png)

**Comment by bshamir** [comment]
It seems good if we add one more section in testimonial.

**Comment by thebuggybug** [comment]
What kind of ? 
Is it mentioned in wireframe ?

**Comment by thebuggybug** [comment]
Hello All, the iissue has been sort out by adding some psedu class CSS to testimonail text. Now there will be inverted Comma and the first letter wll be larger and different in color than entire text. I have attched image of debug. Added this to CSS `.testimonial-text::first-letter{
  font-size: 250%;
  color: #74C042;
}`
![image](https://user-images.githubusercontent.com/68284654/118264553-9672b400-b4d7-11eb-9f3c-411f59d4ebfa.png)
